BasementRenovator.py

```diff
@@ -660,7 +660,16 @@ def tryToPaint(self, event):
             return
         self.lastTile.add((x, y))
 
-        en = Entity(x, y, int(paint.ID), int(paint.variant), int(paint.subtype), 1.0)
+        selection = self.scene().selectedItems()
+        paintID, paintVariant, paintSubtype = paint.ID, paint.variant, paint.subtype
+        if paint.config.hasBitfields and len(selection) == 1:
+            selectedEntity = selection[0]
+            if selectedEntity.entity.config == paint.config:
+                paintID = selectedEntity.entity.Type
+                paintVariant = selectedEntity.entity.Variant
+                paintSubtype = selectedEntity.entity.Subtype
```

Comment:
Interesting, I'm not sure I follow, so if you have an entity selected that's the same as your palette entity when you paint it'll paint it with the same subtype etc.? Either way, can you document this in NicheFeatures.md?

Reply:
Yes. This is helpful when placing a bunch of entities with the same (or even just similar) parameters, like a line of matching fissures. Can add to NicheFeatures.md.